src/register-org/components/before-you-start/before-you-start.component.html

```diff
@@ -0,0 +1,70 @@
+<main id="content" class="govuk-main-wrapper govuk-!-margin-top-4" role="main" tabindex="-1">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        Apply for an organisation to manage civil, family and tribunal cases
+      </h1>
+      <hr>
+      <br>
+      <p class="govuk-body">
+        MyHMCTS is a service for submitting and managing applications. It can be used by organisations and legal professionals.
+      </p>
+      <p class="govuk-body">
+        If you are not sure if your organisation will be able to use this service, see the <a class="govuk-link" href="https://www.gov.uk/guidance/register-for-myhmcts-online-case-management-for-legal-professionals" target="_blank">MyHMCTS guidance on GOV.UK</a>.
+      </p>
+      <div class="govuk-body">
+        <p>By registering, you will:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>create an account for your organisation</li>
+          <li>act as the MyHMCTS administrator for the organisation</li>
+        </ul>
+      </div>
+      <h3 class="govuk-heading-m">Before you start</h3>
+      <p class="govuk-body">
+        MyHMCTS is a service for organisations and legal professionals. Do not use this service if you are a member of the public making an application on your own behalf.
+      </p>
+      <p class="govuk-body">
+        Make sure the information you give us is complete and accurate. If there are any errors in the application, HMCTS will reject it and you will need to start again.
+      </p>
+      <p class="govuk-body">
+        Check if your organisation already has an account. If so, ask your organisation's MyHMCTS administrator to give you access.
+      </p>
+      <p class="govuk-body">
+        Do not create an additional account for your organisation unless you need to keep parts of your business separate.
+      </p>
+      <div class="govuk-body">
+        <p>You will not be able to:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>share cases between accounts</li>
+          <li>add users to more than one organisation account</li>
+          <li>manage both accounts from one place</li>
+        </ul>
+      </div>
+      <div class="govuk-body govuk-!-margin-top-6">
+        <form>
+          <div class="govuk-checkboxes__item">
+            <input class="govuk-checkboxes__input" type="checkbox" id="confirmed-organisation-account" name="confirmedOrganisationAccount" formControlName="confirmedOrganisationAccount">
+            <label class="govuk-label govuk-checkboxes__label" for="confirmed-organisation-account">
+              I've checked whether my organisation already has an account
+            </label>
+          </div>
+          <div class="govuk-body govuk-!-margin-top-6">
+            <button class="govuk-button govuk-buton--primary" [routerLink]="'/register-org-new/company-house-details'">Start</button>
```

Comment:
Don't hardcode the 'register-org-new' part, as this will change later on, and we don't want to go through all these files to change it here as well.

Reply:
I have defined a read-only property in the Register Org service and using it here.
`public readonly REGISTER_ORG_NEW_ROUTE = 'register-org-new';`